<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="display">
          <property name="text">
           <string>DISPLAY</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mode_cb">
          <item>
           <property name="text">
            <string>SERVER</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CLIENT</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="stream_ctrl_lbl">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(143, 240, 164);</string>
             </property>
             <property name="text">
              <string>STREAM CONTROL</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="server_ctrl_lbl">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(249, 240, 107);</string>
             </property>
             <property name="text">
              <string>SERVER CONTROL</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="address_lbl">
           <property name="text">
            <string>IP ADDRESS</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="stream_pb">
             <property name="text">
              <string>START</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="server_pb">
             <property name="text">
              <string>OPEN</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="settings_lbl">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 190, 111);</string>
           </property>
           <property name="text">
            <string>SETTINGS</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="target_adress_lbl">
             <property name="text">
              <string>Target Address:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="target_address_le"/>
           </item>
           <item>
            <widget class="QLabel" name="target_port_lbl">
             <property name="text">
              <string>Target Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="target_port_le"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="control_pb">
             <property name="text">
              <string>PAUSE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="client_pb">
             <property name="text">
              <string>CONNECT</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
